artemis-server/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyAcceptor.java

```diff
@@ -206,6 +207,8 @@
 
    final AtomicBoolean warningPrinted = new AtomicBoolean(false);
 
+   private static int communicationTimeout = 30; //seconds
```

Comment:
This should be named read time out similar to netty naming

Reply:
This also should be config based with defaults declared as per how other bits are configured and defaulted for netty

Reply:
Also shouldn't be static, as you may want to configure different acceptors to different configs

Reply:
I have done a very quick (not 100% complete) example of the way to better config this and how to have default (with disabled), and also how to add specific logger for the other review comment i left.

If you want to look at:

https://github.com/michaelandrepearce/activemq-artemis/commit/4e2026c0fce5a75dcb9551fc4237d3cba9abfb23